src/splitInteger.test.js

```diff
@@ -4,18 +4,18 @@ const splitInteger = require('./splitInteger');
 
 test(`should split a number into equal parts
   if a value is divisible by a numberOfParts`, () => {
-
+  expect(splitInteger(15, 5)).toStrictEqual([3, 3, 3, 3, 3]);
 });
 
 test(`should return a part equals to a value
   when splitting into 1 part`, () => {
-
+  expect(splitInteger(10, 1).length).toBe(1);
```

Comment:
are you sure that this is correct test? Have you run it?

Reply:
I'm not sure this is the right test but I run it and it passed